### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by burningdevil**

### CODE REVIEW

**Comment by burningdevil** [comment]
@burningdevil  dev

**Review by leen1218 — APPROVED** [review]

**Comment by burningdevil** [comment]
@burningdevil a

**Comment by burningdevil** [comment]
@burningdevil  a comment

**Comment by burningdevil** [comment]
@burningdevil a

**Comment by burningdevil** [comment]
@burningdevil  c

**Comment by burningdevil** [comment]
@burningdevil  c

**Comment by burningdevil** [comment]
@burningdevil a

**Comment by burningdevil** [comment]
@burningdevil  asdasd

**Comment by burningdevil** [comment]
@burningdevil  assd

**Comment by burningdevil** [comment]
@oliveshell  asdsad

**Comment by burningdevil** [comment]
@oliveshell  asdasd asdsadasdas sadasdas

**Comment by burningdevil** [comment]
@oliveshell  asdasd asdasdasdas

**Comment by burningdevil** [comment]
@oliveshell assad 222

**Comment by burningdevil** [comment]
@burningdevil  a comment

**Comment by burningdevil** [comment]
@leen1218  a comment

**Comment by burningdevil** [comment]
@burningdevil a comment2

**Comment by burningdevil** [comment]
@burningdevil  a com

**Comment by burningdevil** [comment]
@leen1218  ac

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@
 a
 b
 c
+d
```
Comment (burningdevil) [inline .gitignore:11]: @burningdevil another c

**Comment by burningdevil** [comment]
@oliveshell  a co

**Comment by burningdevil** [comment]
@burningdevil a c

**Comment by burningdevil** [comment]
@oliveshell  aaa

**Comment by burningdevil** [comment]
@burningdevil  abcd

**Pull request closed without merging**